apache-maven/src/bin/mvn

```diff
@@ -196,9 +196,16 @@ export MAVEN_PROJECTBASEDIR
 MAVEN_CMD_LINE_ARGS="$MAVEN_CONFIG $@"
 export MAVEN_CMD_LINE_ARGS
 
+# MNG-8248
+ENABLE_NATIVE_ACCESS=""
+if "$JAVACMD" --enable-native-access=ALL-UNNAMED -version >/dev/null 2>&1; then
```

Comment:
The expectation is that when this does not fail then the JVM access this option?

Reply:
here we check if we can use it ... it is available for JDK 17+

Reply:
OK, this is what I thought.